.env

```diff
@@ -1,3 +1,4 @@
+# We are aware these api keys are public, please do not report them, it is intentional.
```

Comment:
"Thank you for your concern, these api keys are intentionally public. Please do not report them." ?